### ISSUE TITLE
Accessibility: Insufficient focus state for input fields and submit button

### ISSUE THREAD

**Issue by GuySartorelli** [silverstripe/silverstripe-login-forms#230]
The focus state for input fields in the login form is not visible.

The focus state for buttons in the login/logout form doesn't meet contrast requirements.

Liaise with the design team to get appropriate focus states.

Related: https://github.com/silverstripe/.github/issues/443

**Comment by cassiezoest** [comment]
Just noting that the login page changes depending on your OS theme so there's a light theme and dark theme. 

For the light theme, we can stick to a dark blue outline for all elements and a white outline for the dark theme. We will need an offset on the login button as well

**Comment by GuySartorelli** [comment]
Closing as a duplicate of https://github.com/silverstripe/.github/issues/443 - I'll handle it at the same time as that issue.
